schema-registry/serdes/src/main/java/com/hortonworks/registries/schemaregistry/serdes/avro/AvroSerDesHandler.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 Hortonworks.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hortonworks.registries.schemaregistry.serdes.avro;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import org.apache.avro.Schema;
+
+/**
+ * Interface for serializing and deserializing avro payloads.
+ */
+public interface AvroSerDesHandler {
```

Comment:
OK, no real code duplication here but it takes `AvroSerDesHandler` to delegate to. I am not rigid on not having it but trying to avoid any new interface to keep it simple enough. 

I am fine with these changes.